spec/features/card_spec.rb

```diff
@@ -10,13 +10,13 @@
   end
 
   context 'When user try to create card' do
-    it 'not allows to create empty cards' do
+    it 'not allows to create empty cards_mailer' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.